src/constants/featureFlags.ts

```diff
@@ -1,6 +1,10 @@
 export const FEATURE_FLAGS = {
-  HOMEPAGE_TEMPLATES: "homepage_new_templates",
   REPO_PRIVATISATION: "repo_privatisation",
   BANNER: "banner",
   NPS_FORM: "nps_form",
+  ENABLED_BLOCKS: "homepage_enabled_blocks",
 } as const
+
+// NOTE: Only have 4 default blocks:
+// hero/infobar/infopic/resources
+export const NUM_DEFAULT_HOMEPAGE_BLOCKS = 4
```

Comment:
Hmm I am abit confused how this works, should we not list out the default blocks here and have a util function that tells us if this particular block is indeed a default block or not?

Reply:
actually something that i thought about was just to remove the 4 default blocks from the growthbook, but i thought that we might wanna rollback those also. (this is marginal, i don't mind just removing from gb) 

we need a util here that determines if the feature is on, and what happened previously was determining if the flag is on/off. given that the flag no longer exists, i just decided to check if all the default blocks are in (a good enough check()

Reply:
nit: if the code is not intended to rollback default blocks, could we just list out the blocks so that we are explict in the blocks that we support? 


non blocking since code is functional: 

Hmm could we just do away with this? 
the feature flag just contains a list of allowed homepage blocks -> then FE displays blocks to users based on that. 

I cannot reason how our fe behaves if say GB states 

`hero`, `infobar`, `infopic` + say `announcements`. in this case feature flag does not get enabled, and the default 4 blocks are still shown, right?

Reply:
hey apologies if it's unclear - i wanted to also have the ability to rollback default blocks because the drag/drop refactor also went from js -> ts. 

so, for example, if we have the 4 mentioned above, the modal **will not show** but the only blocks that are shown are `hero/infobar/infopic/announcements`. i'm not super sure of the incremental value of adding the ff back cos of additional complexity + idt we will rollback all the blocks until we have less than 4

Reply:
Wait this is not true no? cuz later in the code we check the number of allowed blocks (not the unique identifiers per block) and if === 4, that means that the feature flag is not turned on? did I misunderstand this?

Reply:
actually you're right - i skimmed over this part and did a simple replacement. i'll update so that the check is not a blanket check but `blocks.includes(<feature_name>)`